urllib3/util/connection.py

```diff
@@ -77,7 +86,6 @@ def create_connection(address, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
             err = e
             if sock is not None:
                 sock.close()
-                sock = None
 
     if err is not None:
```

Comment:
Oooh! Sorry Github review comments only show the line your comment was on, I will look at this. :) Thanks!